scripts/install-for-ci.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
-fetch
```

Comment:
I am not sure why this line was here, or if it needs to be here for some reason. If so, I can edit my PR to leave that, and add a comment on when it is used.

@fukamachi, @snmsts ?